pom.xml

```diff
@@ -228,7 +228,7 @@
         <clojure.math.numeric-tower.version>0.0.1</clojure.math.numeric-tower.version>
         <carbonite.version>1.5.0</carbonite.version>
         <snakeyaml.version>1.11</snakeyaml.version>
-        <httpclient.version>4.3.3</httpclient.version>
+        <httpclient.version>4.5</httpclient.version>
```

Comment:
This is needed for storm-opentsdb. It is good to have this bump for other modules as it has improvements than existing version.